webpages/settings/components/addon-body.js

```diff
@@ -30,6 +30,12 @@ export default async function ({ template }) {
       addonSettings() {
         return this.$root.addonSettings[this.addon._addonId];
       },
+      showUpdateNotice() {
+        if (!this.addon.latestUpdate || !this.addon.latestUpdate.temporaryNotice) return false;
+        const [extMajor, extMinor, _] = this.$root.version.split(".");
```

Comment:
Why not just not destructure the unused variables?

Reply:
The same code is already used in `index.js` for the "New" tag and in another function in the same file for the "New settings" tag. I think it would make sense to move it into a module and make any changes there.